spec/index.bs

```diff
@@ -5127,7 +5127,7 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                         <div class=validusage>
                             - |this| is [=valid=].
                             - The {{GPUBindGroupLayoutEntry/binding}} of each entry in |descriptor| is unique.
-                            - The {{GPUBindGroupLayoutEntry/binding}} of each entry in |descriptor| must be &lt; 65536.
+                            - The {{GPUBindGroupLayoutEntry/binding}} of each entry in |descriptor| must be &lt; 640.
```

Comment:
IIRC @kainino0x asked for a limit to be added for this value.

Reply:
Yes, I think we should, although it doesn't have to be in the same PR! We can leave the linked issue open until that's resolved.